build.gradle

```diff
@@ -37,7 +53,7 @@ dependencies {
     compile 'io.swagger:swagger-annotations:1.5.13'
     compile 'com.fasterxml.jackson.core:jackson-annotations'
     compile 'com.fasterxml.jackson.core:jackson-databind'
-    compile 'javax.validation:validation-api'
+    compile 'javax.validation:validation-api:2.0.1.Final'
```

Comment:
we have https://github.com/reportportal/commons-bom
check it out and it s better probably good idea to move validation api dependency there

Reply:
move versions in props